<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room_DataWid</class>
 <widget class="QWidget" name="Room_DataWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="titleWid" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="volWid" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="curWid" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="powWid" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="pfWid" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
